### DISCUSSION TITLE
Add feature to tie `hidden` to `x-show` so that taildwindcss space-between and divide-between utilities?

### DISCUSSION THREAD

**Discussion by jasikpark** [alpinejs/alpine#929] [4. General]
https://tailwindcss.com/docs/upgrading-to-v2#add-hidden-to-any-template-tags-within-space-or-divide-elements 

In tailwindcss 2.0 they use the `hidden` attribute to determine whether an element should have padding for space-between and divide-between.

Does it make sense to tie the attributes together so that you don't need 
<div x-show="hello.world.this.is.a.deep.object" :hidden="hello.world.this.is.a.deep.object"></div>

Idk whether this is something to support?

**Comment by DanielDarrenJones** [comment]
This would be really helpful to help reduce the amount of bindings we need to add and simplify our code.
